test/travis.yml

```diff
@@ -67,9 +69,9 @@ jobs:
       env: CH_BUILDER=docker
 
     - <<: *stage_install
-      env: CH_BUILDER=buildah-runc    TARBALL=export       INSTALL=yes
+      env: CH_BUILDER=buildah         TARBALL=export       INSTALL=yes
     - <<: *stage_install
-      env: CH_BUILDER=buildah-setuid  TARBALL=export       INSTALL=yes
+      env: CH_BUILDER=buildah-runc    TARBALL=export       INSTALL=yes
```

Comment:
Did you want buildah-setuid to also be included in the install stage?

Reply:
Good catch. No, because Travis does at most five jobs in parallel, so I'm trying to keep the stages to five jobs wide so they complete in one interval rather than two. Also, the Buildah tests take a lot longer.